src/org/jitsi/impl/neomedia/rtcp/RTCPTCCPacket.java

```diff
@@ -0,0 +1,325 @@
+                    // than what we handle above. As per the draft, if we want
+                    // send feedback with such deltas, we should split it up
+                    // into multiple RTCP packets. We can't do that here in the
+                    // constructor.
+                    throw new IllegalArgumentException("Delta too big, needs new reference.");
+                }
+
+                // If the packet was received, the next delta will be relative
+                // to its time. Otherwise, we'll just the previous reference.
+                nextReferenceTime = ts;
+            }
+
+            int symbolOffset;
+            switch (seqDelta % 7)
+            {
+            case 0:
+            case 4:
+                symbolOffset = 4;
+                break;
+            case 1:
+            case 5:
+                symbolOffset = 2;
+                break;
+            case 2:
+            case 6:
+                symbolOffset = 0;
+                break;
+            case 3:
+            default:
+                symbolOffset = 6;
+            }
+
+            symbol <<= symbolOffset;
+            buf[off] |= symbol;
+        }
+
+        off++;
+        if (packetCount % 7 <= 3)
+        {
+            // the last chunk was not complete
+            buf[off++] = 0;
+        }
+
+
+        fci = new byte[off + deltaOff];
+        System.arraycopy(buf, 0, fci, 0, off);
+        System.arraycopy(deltas, 0, fci, off, deltaOff);
+        this.packets = packets;
+    }
+
+    /**
+     *
+     * @param next
+     * @return
+     */
+    /*
+    public static Map<Integer, Long> getPackets(ByteArrayBuffer next)
+    {
+        Map<Integer,Long> packets = new TreeMap<>(RTPUtils.sequenceNumberComparator);
+        ByteArrayBuffer fciBuffer = getFCI(next);
+        if (fciBuffer == null)
+        {
+            return packets;
+        }
+
+        byte[] fci = fciBuffer.getBuffer();
+        int off = fciBuffer.getOffset(), len = fciBuffer.getLength();
+
+        for (int i = 0; i < (len / 4); i++)
+        {
+            int pid = (0xFF & fci[off + i * 4 + 0]) << 8 | (0xFF & fci[off + i * 4 + 1]);
+            packets.add(pid);
+
+            // First byte of the BLP
+            for (int j = 0; j < 8; j++)
+                if (0 != (fci[off + i * 4 + 2] & (1 << j)))
+                    packets.add((pid + 1 + 8 + j) % (1 << 16));
+
+            // Second byte of the BLP
+            for (int j = 0; j < 8; j++)
+                if (0 != (fci[off + i * 4 + 3] & (1 << j)))
+                    packets.add((pid + 1 + j) % (1 << 16));
+        }
+
+        return packets;
+    }
+    */
+
+    /**
+     * @return
+     */
+    synchronized public Map<Integer, Long> getPackets()
+    {
+        if (packets == null)
+        {
+            // parse this.fci as containing NACK entries and initialize
+            // this.packets
+            //packets = getPackets(new RawPacket(fci, 0, fci.length));
```

Comment:
Yeah, I'm sorry I should have mentioned -- that part will be implemented shortly (I hope).